### DISCUSSION TITLE
[A-L] (18 maggio 2024) Esercizio 5

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/progettazione-di-algoritmi#99] [Esercitazioni A-L]
![immagine](https://github.com/user-attachments/assets/3f6c37b9-00c4-43e1-9873-938aac312d9c)

**Comment by daromacarlo** [comment]
Esercizio svolto con tempo di esecuzione O(n^2)
![immagine](https://github.com/user-attachments/assets/acf23773-0a8d-4547-a36b-fbbf72e6713c)
![Immagine 2025-02-05 121250](https://github.com/user-attachments/assets/076c5b84-eb3a-4b50-86db-fc8d9625507e)

Versione per ritornare valori massimi
![immagine](https://github.com/user-attachments/assets/6eb991a0-e8f8-487a-829f-8de62a761ee9)
![immagine](https://github.com/user-attachments/assets/07b4e0b8-f7cf-4dfc-a8a0-4ff9ca77a0d0)

Versione in O(n)
![immagine](https://github.com/user-attachments/assets/d787cf94-3ca6-4d79-9be6-88ae60a5b013)
![immagine](https://github.com/user-attachments/assets/10b7591e-205d-40b9-a14f-fd3fd9870e46)
